packages/core/lib/commands/command-options.js

```diff
@@ -0,0 +1,13 @@
+const options = [
+    {
+      option: "--network <name>",
+      description:
+        "Specify the network to use. Network name must exist in the configuration."
+    },
+    {
+      option: "--config <file>",
+      description:
+        "Specify configuration file to be used. The default is truffle-config.js"
+    }
+];
+ module.exports = options;
```

Comment:
let's move this file outside of `lib/commands`, since that directory is only actual commands right now

Reply:
moved!